content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md

```diff
@@ -12,11 +12,11 @@ This requires a 2-step process:
 
 ## Set up an access token
 
-Iterative Studio uses access tokens to authorize [DVCLive] to send live updates
-to the metrics and plots. The access token must be present in any request that
-sends data to the Iterative Studio ingestion endpoint. Requests with missing or
-incorrect access tokens are rejected with an appropriate HTTP error code and
-error message.
+Iterative Studio uses access tokens to authorize DVC and [DVCLive] to send
+experiments and live updates to the metrics and plots. The access token must be
+present in any request that sends data to the Iterative Studio ingestion
+endpoint. Requests with missing or incorrect access tokens are rejected with an
+appropriate HTTP error code and error message.
```

Comment:
Now that the access token is used by dvc too, I think we need to rewrite this page or extract it out.

Reply:
That is a good point.
Probably outside the scope of the P.R. and better to be discussed in https://github.com/iterative/dvc.org/issues/4440

I would perhaps not touch this introduction for now and leave the page to be only about `live updates`

Reply:
I think we should at least mention "to send experiments" here as we are redirecting here for setup instructions. Let me know what you think (that's the only change here btw).

Reply:
> I think we should at least mention "to send experiments" here as we are redirecting here for setup instructions. Let me know what you think (that's the only change here btw).

I am ok with merging this as it is.

> we are redirecting here for setup instructions

As you mentioned, what we probably need to do is to extract the info, for example having something like a `connect to Studio` page that doesn't get into the downstream use case (`live updates`, `exp push`, etc.) that we can cross reference

Reply:
Looks okay to me for now. I'm a little hesitant to say we need another page just for how to setup an access token. Feels like it creates more friction for live sharing. @tapadipti PTAL and add any thoughts you have.

Reply:
@tapadipti Do we have `any of the other env variables mentioned in this guide? For customers who have on-prem Studio, how do they know how to configure live metrics and plots?